portfolio/src/main/java/com/google/sps/servlets/LogInServlet.java

```diff
@@ -20,10 +20,14 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     Boolean loggedIn;
     String email = null;
     String url;
+    Boolean access = false;
 
     if (userService.isUserLoggedIn()) {
       loggedIn = true;
       email = userService.getCurrentUser().getEmail();
+      if (email.equals("shershi@google.com") || email.equals("sherryshi2001@gmail.com") || email.equals("ricazhang@google.com") || email.equals("alfredh@google.com")) {
+        access = true;
+      }
```

Comment:
I would suggest storing a list of strings and check if the email is present in the list.

Reply:
okay, changed!